### PR TITLE
zh-translation: zh/docs/ops/configuration/traffic-management/dns-proxy

### PR DESCRIPTION
**Pull request by nicole-lihui**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+除了捕获应用流量，Istio 还可以捕获 DNS 请求，以提高网格的性能和可用性。
+当 Istio 代理 DNS 时，所有来自应用程序的 DNS 请求将会被重定向到 Sidecar，因为 Sidecar 存储了域名到 IP 地址的映射。如果请求被 Sidecar 处理，它将直接给应用返回响应，避免了对上游DNS服务器的往返。反之，请求将按照标准的`/etc/resolv.conf` DNS 配置向上游转发。
+
+虽然 Kubernetes 为 Kubernetes `Service` 提供了一个开箱即用的 DNS 解析，但任何自定义的  `ServiceEntry` 都不会被识别。有了这个功能，`ServiceEntry` 地址可以被解析，而不需要自定义 DNS 服务配置。对于 Kubernetes `Service` 来说，一样的 DNS 响应，但减少了 `kube-dns` 的负载，并且提高了性能。
+
+该功能也适用于在 Kubernetes 外部运行的服务。这意味着所有的内部服务都可以被解析，而不需要再使用笨重的运行方法来暴露集群外的 Kubernetes DNS 条目。
+
+## 开始{#getting-started}
+
+此功能默认情况下未启用。要启用该功能，请在安装 Istio 时使用以下设置:
```
Comment (orangegzx) [inline content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md:19]:
```suggestion
此功能默认情况下未启用。要启用该功能，请在安装 Istio 时使用以下设置：
```

File: content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+{{< /text >}}
+
+现在，发送一个请求：
+
+{{< text bash >}}
+$ curl -v auto.internal
+*   Trying 240.240.0.1:80...
+{{< /text >}}
+
+您可以看到，请求被发送到一个自动分配的地址  `240.240.0.1` 上。这些地址将从 `240.240.0.0/16` 预留的 IP 地址池中挑选出来，以避免与真实的服务发生冲突。
```
Comment (orangegzx) [inline content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md:109]:
```suggestion
您可以看到，请求被发送到一个自动分配的地址 `240.240.0.1` 上。这些地址将从 `240.240.0.0/16` 预留的 IP 地址池中挑选出来，以避免与真实的服务发生冲突。
```

File: content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  - 198.51.100.0
+  hosts:
+  - address.internal
+  ports:
+  - name: http
+    number: 80
+    protocol: HTTP
+{{< /text >}}
+
+除了 DNS 捕获，去 `address.internal` 的请求可能解析失败。一旦启用，您将收到一个基于 `address` 配置的响应：
```
Comment (kebe7jun) [inline content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md:62]:
```suggestion
如果不开启 DNS 代理功能，请求 `address.internal` 时可能解析失败。一旦启用，您将收到一个基于 `address` 配置的响应：
```

File: content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
+---
+title: DNS Proxying
```
Comment (kebe7jun) [inline content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md:2]: 翻译吧，叫 DNS 代理。

File: content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+description: How to configure DNS proxying.
```
Comment (kebe7jun) [inline content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md:3]: 如何配置使用 DNS 代理。

File: content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+除了 DNS 捕获，去 `address.internal` 的请求可能解析失败。一旦启用，您将收到一个基于 `address` 配置的响应：
+
+{{< text bash >}}
+$ curl -v address.internal
+*   Trying 198.51.100.1:80...
+{{< /text >}}
+
+## 自动分配地址{#address-auto-allocation}
+
+在上面的示例中，对于发送请求的服务，您有一个预定义的IP地址。但是没有一个稳定的地址去访问外部服务是很正常的，因此需要 DNS。在这种情况下，DNS 代理没有足够的信息去返回一个响应，这将需要向上游转发 DNS 请求。
```
Comment (kebe7jun) [inline content/zh/docs/ops/configuration/traffic-management/dns-proxy/index.md:71]: 这段读起来不是很通顺，你整理一下。

**Review by kebe7jun — APPROVED** [review]
lgtm

**Pull request merged**